NullPointerException from using :include-image: on svg files.

I exported an image from nomnoml.com/ (I like the UML diagrams better than PlantUML) and tried to import the svg file.

```
:include-image: uml-nomnoml.svg {fit: true}
```

Znai threw an exception with the cause:

```
Caused by: java.lang.NullPointerException: Cannot invoke "java.awt.image.BufferedImage.getWidth()" because "bufferedImage" is null
    at org.testingisdocumenting.znai.extensions.image.ImagePluginBase.setWidthHeight (ImagePluginBase.java:142)
    at org.testingisdocumenting.znai.extensions.image.ImagePluginBase.process (ImagePluginBase.java:127)
    at org.testingisdocumenting.znai.extensions.image.ImageIncludePlugin.process (ImageIncludePlugin.java:73)
    at org.testingisdocumenting.znai.extensions.TrackingIncludePlugin.process (TrackingIncludePlugin.java:65)
    at org.testingisdocumenting.znai.parser.commonmark.MarkdownVisitor.handleIncludePlugin (MarkdownVisitor.java:217)
    at org.testingisdocumenting.znai.parser.commonmark.MarkdownVisitor.visit (MarkdownVisitor.java:145)
    at org.testingisdocumenting.znai.parser.commonmark.include.IncludeBlock.accept (IncludeBlock.java:43)
    at org.commonmark.node.AbstractVisitor.visitChildren (AbstractVisitor.java:137)
    at org.commonmark.node.AbstractVisitor.visit (AbstractVisitor.java:28)
    at org.commonmark.node.Document.accept (Document.java:7)
    at org.testingisdocumenting.znai.parser.commonmark.MarkdownParser.parsePartial (MarkdownParser.java:89)
    at org.testingisdocumenting.znai.parser.commonmark.MarkdownParser.parse (MarkdownParser.java:61)
    at org.testingisdocumenting.znai.website.WebSite.parseMarkupAndUpdateTocItemAndSearch (WebSite.java:533)
```

I don't *need* svg support. I was able to work around the issue since nomnoml also exports png files. I think Znai should at least throw a helpful error message in this case.

--- a/znai-core/src/main/java/org/testingisdocumenting/znai/parser/docelement/DocElementCreationParserHandler.java
+++ b/znai-core/src/main/java/org/testingisdocumenting/znai/parser/docelement/DocElementCreationParserHandler.java
@@ -285,13 +285,19 @@ public class DocElementCreationParserHandler implements ParserHandler {
             AuxiliaryFile auxiliaryFile = resourcesResolver.runtimeAuxiliaryFile(destination);
             BufferedImage image = resourcesResolver.imageContent(destination);
 
-            append(DocElementType.IMAGE, "title", title,
-                    "destination", docStructure.fullUrl(auxiliaryFile.getDeployRelativePath().toString()),
-                    "alt", alt,
-                    "inlined", true,
-                    "timestamp", componentsRegistry.timeService().fileModifiedTimeMillis(auxiliaryFile.getPath()),
-                    "width", image.getWidth(),
-                    "height", image.getHeight());
+            Map<String, Object> props = new HashMap<>();
+            props.put("title", title);
+            props.put("destination", docStructure.fullUrl(auxiliaryFile.getDeployRelativePath().toString()));
+            props.put("alt", alt);
+            props.put("inlined", true);
+            props.put("timestamp", componentsRegistry.timeService().fileModifiedTimeMillis(auxiliaryFile.getPath()));
+
+            if (image != null) {
+                props.put("width", image.getWidth());
+                props.put("height", image.getHeight());
+            }
+
+            append(DocElementType.IMAGE, props);
 
             auxiliaryFiles.add(auxiliaryFile);
         } else {

--- a/znai-core/src/test/groovy/org/testingisdocumenting/znai/parser/MarkdownParserTest.groovy
+++ b/znai-core/src/test/groovy/org/testingisdocumenting/znai/parser/MarkdownParserTest.groovy
@@ -313,6 +313,16 @@ world""")
                             type: 'Image']]
     }
 
+    @Test
+    void "standalone svg image"() {
+        TEST_COMPONENTS_REGISTRY.timeService().fakedFileTime = 200000
+
+        parse("![alt text](images/test.svg \"custom title\")")
+        content.should == [[title: "custom title", destination: '/test-doc/test.svg',
+                            alt: 'alt text', inlined: false,
+                            timestamp: 200000,
+                            type: 'Image']]
+    }
 
     @Test
     void "image with external ref"() {

--- a/znai-core/src/test/resources/images/test.svg
+++ b/znai-core/src/test/resources/images/test.svg
@@ -1,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36 36" >
+    <path fill="#BDDDF4" d="M13 3H7l-7 9h10z"/>
+    <path fill="#5DADEC" d="M36 12l-7-9h-6l3 9z"/>
+    <path fill="#4289C1" d="M26 12h10L18 33z"/>
+    <path fill="#8CCAF7" d="M10 12H0l18 21zm3-9l-3 9h16l-3-9z"/>
+    <path fill="#5DADEC" d="M18 33l-8-21h16z"/>
+</svg>

--- a/znai-docs/znai/release-notes/1.63/fix-2023-02-21-image-svg-null-pointer.md
+++ b/znai-docs/znai/release-notes/1.63/fix-2023-02-21-image-svg-null-pointer.md
@@ -1,0 +1,1 @@
+* Fix: [Images](visuals/images) null pointer when using with SVGs

--- a/znai-website-gen/src/main/java/org/testingisdocumenting/znai/extensions/image/ImagePluginBase.java
+++ b/znai-website-gen/src/main/java/org/testingisdocumenting/znai/extensions/image/ImagePluginBase.java
@@ -137,6 +137,10 @@ abstract class ImagePluginBase implements Plugin {
 
     private void setWidthHeight(BufferedImage bufferedImage,
                                 Map<String, Object> props) {
+        if (bufferedImage == null) {
+            return;
+        }
+
         Double pixelRatio = pixelRatio();
 
         props.put("width", bufferedImage.getWidth() / pixelRatio);

